docs/cloudlinux-os-plus/README.md

```diff
@@ -1429,35 +1429,49 @@ Use this logging with caution because when it is enabled, the size of the daemon
 
 ## AccelerateWP
 
-### Overview
+### Get started
 
 AccelerateWP carries a suite of optimization features that can be enabled and automatically configured for the end-user's site.
 
 There are AccelerateWP and AccelerateWP Premium feature suites.
 
-#### AccelerateWP suite
-This is a basic suite which includes [AccelerateWP base feature](https://user-docs.cloudlinux.com/wpos-plugin/#acceleratewp-feature-wordpress-optimization-plugin): a WordPress optimization plugin that provides full page caching, GZIP compression and some other useful optimizations.
+AccelerateWP suite is always enabled when AccelerateWP is turned on. Choose whether you want to offer AccelerateWP Premium 
+for your users and click "Turn on" to start exploring AccelerateWP. 
 
-#### AccelerateWP Premium suite
-:::tip Note
-If you'd like to try AccelerateWP Premium suite, you should participate in the Beta tester program. To become a beta tester, please send your request at our Beta program page with the signup form [here](https://www.cloudlinux.com/wp-performance/). Once you submit the request, we will send you a confirmation email with program details, terms of use, and installation instructions.
-:::
+![](./images/awp/AWPGettingStarted.png)
 
-This is a premium suite which includes [Object Caching feature](https://user-docs.cloudlinux.com/wpos-plugin/#acceleratewp-premium-object-caching-feature).
+Once the AccelerateWP suite is enabled by administrator,
+end-users will see an AccelerateWP tab in their cPanel interface and be able to activate the optimization feature.
+
+When AccelerateWP Premium suite enabled by administrator, 
+end-users will see the Object Caching feature in their interface, 
+but cannot install the plugin unless they purchase the feature using [WHMCS or 3'd party billing](#billing-integration).
 
-The Object Caching mechanism stores database query results in additional storage for quick access. This mechanism is really helpful in case if website needs to process multiple pages per second as requests come in and may be helpful in case when full page caching cannot be used, e.g. on personalized pages.
+#### AccelerateWP suite
+This is a basic suite which includes [AccelerateWP base feature](https://user-docs.cloudlinux.com/wpos-plugin/#acceleratewp-feature-wordpress-optimization-plugin): 
+a WordPress optimization plugin that provides full page caching, GZIP compression and some other useful optimizations.
+
+#### AccelerateWP Premium suite
 
 :::warning Attention
 This feature is free of charge only during the beta testing period. Afterwards it will be charged.
```

Comment:
Should we add a concrete date here? April 2023

Reply:
I'm not a fan of concrete dates, so I added the same message that we have in lvemanager:

> Beta testing period ends no earlier than March 2023.